### PR TITLE
bugfix-280

### PR DESCRIPTION
**Pull request by wussup**
Check if the innerListener_ is an array and loop if it is

### CODE REVIEW

File: src/olgm/listener/PropertyListener.js [hunk 1]
```diff
@@ -17,7 +17,11 @@ class PropertyListener extends Listener {
   constructor(target, oldTarget, key, listen) {
     super(target.on('change:' + key, e => {
       if (this.innerListener_) {
-        this.innerListener_.unlisten();
+        if (Array.isArray(this.innerListener_)) {
+          this.innerListener_.forEach(listener => listener.unlisten());
```
Comment (adube) [inline src/olgm/listener/PropertyListener.js:21]: Let's empty the array after unlistening the events, i.e. let's add:

```
this.innerListener_.length = 0;
```

**Review by adube — CHANGES_REQUESTED** [review]
Code is clean.  Let's add a small something (see below)

File: src/olgm/listener/PropertyListener.js (hunk 1 above)
Comment (wussup) [reply src/olgm/listener/PropertyListener.js:21]: A few lines below there is assigned a new object for this.innerListener_. IMHO, it is not needed.
`this.innerListener_ = listen(e.target.get(e.key), e.oldValue);`

> Let's empty the array after unlistening the events, i.e. let's add:
> 
> ```
> this.innerListener_.length = 0;
> ```

**Review by adube — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**